static/change_mysql_pass.sh

```diff
@@ -3,13 +3,13 @@
 # Tech and Me, ©2016 - www.techandme.se
 
 SHUF=$(shuf -i 17-20 -n 1)
-NEWMYSQLPASS=$(cat /dev/urandom | tr -dc "a-zA-Z0-9@#*=" | fold -w $SHUF | head -n 1)
+NEWMYSQLPASS=$(cat < /dev/urandom | tr -dc "a-zA-Z0-9@#*=" | fold -w "$SHUF" | head -n 1)
```

Comment:
Nope said it should be this way, thought it was strange to, but after changing the error went away so, we'll see.